### PR TITLE
TOOLS-2461 upgrade Manta/Triton 19.[12] components to 19.4

### PR DESCRIPTION
**Pull request by timfoster**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -22,77 +22,60 @@ pipeline {
...
-        }
-        stage('minimal-64 19.2.0') {
-            agent {
-                node {
-                    label joyCommonLabels(image_ver: '19.2.0')
-                }
-            }
-            steps {
-                sh('''
-./tools/build_jenkins -u 7f4d80b4-9d70-11e9-9388-6b41834cbeeb -p $MIN_PLATFORM_STAMP
```
[the remark this answers is not in the document]
Comment (timfoster) [reply Jenkinsfile:81]: Yeah, I'm looking to completely remove all support for 19.1 and 19.2 build machines.

With TOOLS-2449 I'm bumping /opt/tools on all Jenkins agents, which means building new agent images, which means updating validate-buildenv.sh, and then bumping deps/eng everywhere, so I figured now would be a great time to reduce the set of supported build environments.

I think we might be able to remove 2018Q1 pretty easily now too, but it's a bigger lift to do 2018Q4 unfortunately..

[the remark this answers is not in the document]
Comment (trentm) [reply Jenkinsfile:81]: > but it's a bigger lift to do 2018Q4 unfortunately..

Yup:

```
electric-moray             a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
manta-garbage-collector    a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
manta-storinfo             a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
muppet                     a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
sdc-amonredis              a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
sdc-assets                 a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
sdc-booter                 a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
sdc-cnapi                  a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
sdc-imgapi                 a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
sdc-manta                  a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
sdc-napi                   a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
sdc-nat                    a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
sdc-papi                   a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
sdc-portolan               a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
sdc-sapi                   a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
sdc-volapi                 a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
triton-cmon                a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
triton-grafana             a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
triton-mockcloud           a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
triton-prometheus          a9368831-958e-432d-a031-f8ce6768d190 triton-origin-x86_64-18.4.0
```

And I'm not sure about mantav1 branches.

[the remark this answers is not in the document]
Comment (trentm) [reply Jenkinsfile:81]: manta-mako#mantav1 still uses 18.1.0 unfortunately: https://github.com/joyent/manta-mako/blob/mantav1/Makefile#L56

**Comment by timfoster** [comment]
The last remaining component using 19.1 (rust-fast) has moved off it, so this is ready for review now.

**Comment by timfoster** [comment]
Sorry about forgetting the 18.1 gunk that was still in nodeconfigs.json - gone now.

**Pull request merged**